export kubeconfig causes nil pointer

**Describe the bug**
Trying to change namespace fails if export KUBECONFIG is used, it's not taken into account at all:

```console
➜  k8ssandra git:(helm_charts) ✗ export KUBECONFIG="/Users/michael.burman/projects/k8ssandra/kubeconfig"
➜  k8ssandra git:(helm_charts) ✗ kubectl get all                                                                       
NAME                 TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
service/kubernetes   ClusterIP   10.96.0.1    <none>        443/TCP   14m
➜  k8ssandra git:(helm_charts) ✗ kubecm ns
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x30 pc=0x103c53f78]

goroutine 1 [running]:
github.com/sunny0826/kubecm/cmd.(*NamespaceCommand).runNamespace(0x0?, 0x0?, {0x10540d600, 0x0, 0x0})
        github.com/sunny0826/kubecm/cmd/namespace.go:44 +0x68
github.com/sunny0826/kubecm/cmd.(*NamespaceCommand).Init.func1(0x1400047d900?, {0x10540d600?, 0x4?, 0x103c5e56c?})
        github.com/sunny0826/kubecm/cmd/namespace.go:32 +0x38
github.com/spf13/cobra.(*Command).execute(0x14000654000, {0x10540d600, 0x0, 0x0})
        github.com/spf13/cobra@v1.7.0/command.go:940 +0x66c
github.com/spf13/cobra.(*Command).ExecuteC(0x1400039e600)
        github.com/spf13/cobra@v1.7.0/command.go:1068 +0x320
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.7.0/command.go:992
main.main()
        github.com/sunny0826/kubecm/main.go:30 +0x24
➜  k8ssandra git:(helm_charts) ✗
```

As you see, kubectl correctly reads the KUBECONFIG setting, but kubecm does not and tries to probably connect to somewhere which isn't running.

**To Reproduce**
Start a kind cluster with different kubeconfig & context than the default. export kubeconfig and try to run kubecm ns.

**Expected behavior**
namespaces should be listed correctly, just like kubectl works. 

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS Sonoma

**Smartphone (please complete the following information):**

**Additional context**

--- a/.all-contributorsrc
+++ b/.all-contributorsrc
@@ -185,6 +185,16 @@
       "contributions": [
         "infra"
       ]
+    },
+    {
+      "login": "burmanm",
+      "name": "Michael Burman",
+      "avatar_url": "https://avatars.githubusercontent.com/u/562574?v=4",
+      "profile": "https://github.com/burmanm",
+      "contributions": [
+        "bug",
+        "code"
+      ]
     }
   ],
   "contributorsPerLine": 7,

--- a/.github/workflows/codeql-analysis.yml
+++ b/.github/workflows/codeql-analysis.yml
@@ -44,6 +44,11 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v4
 
+    - name: Install Go
+      uses: actions/setup-go@v4
+      with:
+        go-version-file: go.mod
+
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
       uses: github/codeql-action/init@v3

--- a/README.md
+++ b/README.md
@@ -128,6 +128,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/jsfaint"><img src="https://avatars.githubusercontent.com/u/571829?v=4?s=100" width="100px;" alt="Jia Sui"/><br /><sub><b>Jia Sui</b></sub></a><br /><a href="https://github.com/sunny0826/kubecm/commits?author=jsfaint" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/AdamRussak"><img src="https://avatars.githubusercontent.com/u/31593022?v=4?s=100" width="100px;" alt="AdamRussak"/><br /><sub><b>AdamRussak</b></sub></a><br /><a href="https://github.com/sunny0826/kubecm/commits?author=AdamRussak" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="http://chenrui.dev"><img src="https://avatars.githubusercontent.com/u/1580956?v=4?s=100" width="100px;" alt="Rui Chen"/><br /><sub><b>Rui Chen</b></sub></a><br /><a href="#infra-chenrui333" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/burmanm"><img src="https://avatars.githubusercontent.com/u/562574?v=4?s=100" width="100px;" alt="Michael Burman"/><br /><sub><b>Michael Burman</b></sub></a><br /><a href="https://github.com/sunny0826/kubecm/issues?q=author%3Aburmanm" title="Bug reports">🐛</a> <a href="https://github.com/sunny0826/kubecm/commits?author=burmanm" title="Code">💻</a></td>
     </tr>
   </tbody>
   <tfoot>

--- a/cmd/namespace.go
+++ b/cmd/namespace.go
@@ -41,6 +41,10 @@ func (nc *NamespaceCommand) runNamespace(command *cobra.Command, args []string) 
 	if err != nil {
 		return err
 	}
+	if len(config.Contexts) == 0 {
+		return fmt.Errorf("no valid context found")
+	}
+
 	currentContext := config.CurrentContext
 	contNs := config.Contexts[currentContext].Namespace
 	namespaceList, err := GetNamespaceList(contNs)

--- a/docs/en-us/contribute.md
+++ b/docs/en-us/contribute.md
@@ -80,6 +80,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/jsfaint"><img src="https://avatars.githubusercontent.com/u/571829?v=4?s=100" width="100px;" alt="Jia Sui"/><br /><sub><b>Jia Sui</b></sub></a><br /><a href="https://github.com/sunny0826/kubecm/commits?author=jsfaint" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/AdamRussak"><img src="https://avatars.githubusercontent.com/u/31593022?v=4?s=100" width="100px;" alt="AdamRussak"/><br /><sub><b>AdamRussak</b></sub></a><br /><a href="https://github.com/sunny0826/kubecm/commits?author=AdamRussak" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="http://chenrui.dev"><img src="https://avatars.githubusercontent.com/u/1580956?v=4?s=100" width="100px;" alt="Rui Chen"/><br /><sub><b>Rui Chen</b></sub></a><br /><a href="#infra-chenrui333" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/burmanm"><img src="https://avatars.githubusercontent.com/u/562574?v=4?s=100" width="100px;" alt="Michael Burman"/><br /><sub><b>Michael Burman</b></sub></a><br /><a href="https://github.com/sunny0826/kubecm/issues?q=author%3Aburmanm" title="Bug reports">🐛</a> <a href="https://github.com/sunny0826/kubecm/commits?author=burmanm" title="Code">💻</a></td>
     </tr>
   </tbody>
   <tfoot>

--- a/docs/zh-cn/contribute.md
+++ b/docs/zh-cn/contribute.md
@@ -37,6 +37,7 @@
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/jsfaint"><img src="https://avatars.githubusercontent.com/u/571829?v=4?s=100" width="100px;" alt="Jia Sui"/><br /><sub><b>Jia Sui</b></sub></a><br /><a href="https://github.com/sunny0826/kubecm/commits?author=jsfaint" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/AdamRussak"><img src="https://avatars.githubusercontent.com/u/31593022?v=4?s=100" width="100px;" alt="AdamRussak"/><br /><sub><b>AdamRussak</b></sub></a><br /><a href="https://github.com/sunny0826/kubecm/commits?author=AdamRussak" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="http://chenrui.dev"><img src="https://avatars.githubusercontent.com/u/1580956?v=4?s=100" width="100px;" alt="Rui Chen"/><br /><sub><b>Rui Chen</b></sub></a><br /><a href="#infra-chenrui333" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/burmanm"><img src="https://avatars.githubusercontent.com/u/562574?v=4?s=100" width="100px;" alt="Michael Burman"/><br /><sub><b>Michael Burman</b></sub></a><br /><a href="https://github.com/sunny0826/kubecm/issues?q=author%3Aburmanm" title="Bug reports">🐛</a> <a href="https://github.com/sunny0826/kubecm/commits?author=burmanm" title="Code">💻</a></td>
     </tr>
   </tbody>
   <tfoot>

